src/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,179 @@
+import { Box, Typography, Link } from "@mui/material";
+import React from "react";
+import SearchBar from "../Common/SearchBar";
+import SuggestedUserItem from "./SuggestedUsers";
+
+const styles = {
+  ad: {
+    height: "100%",
+    width: "100%",
+  },
+  adSpaceContainer: {
+    alignItems: "center",
+    backgroundColor: "gray.light",
+    border: "5px solid",
+    borderColor: "primary.main",
+    borderRadius: 5,
+    boxSizing: "border-box",
+    cursor: "pointer",
+    display: "flex",
+    height: "30%",
+    justifyContent: "center",
+    marginTop: 0,
+    overflow: "hidden",
+    width: "100%",
+  },
+  adTitle: {
+    color: "primary.main",
+    fontWeight: "bold",
+    textAlign: "center",
+  },
+  footerContainer: {
+    height: "5%",
+    width: "100%",
+  },
+  footerItem: {
+    color: "gray.main",
+    paddingRight: "0.75rem",
+  },
+  footerItemText: {
+    fontSize: "0.75rem",
+  },
+  footerNav: {
+    display: "flex",
+    flexWrap: "wrap",
+    height: "100%",
+    paddingX: 2,
+  },
+  headerTitle: {
+    fontWeight: "bold",
+    paddingX: 2,
+    paddingTop: 1,
+  },
+  rightContent: {
+    alignItems: "center",
+    display: "flex",
+    flexDirection: "column",
+    gap: 2,
+    height: "100vh",
+    paddingLeft: "40px",
+    position: "fixed",
+    width: "310px",
+  },
+  searchBarContainer: {
+    height: "5%",
+    width: "100%",
+  },
+  suHeader: {
+    width: "100%",
+  },
+  suList: {
+    display: "flex",
+    flexDirection: "column",
+    flex: 1,
+  },
+  suggestedUserContainer: {
+    backgroundColor: "gray.light",
+    borderRadius: 5,
+    display: "flex",
+    flexDirection: "column",
+    height: "38%",
+    width: "100%",
+    overflow: "hidden",
+  },
+};
+
+const SideBar = () => {
+  return (
+    <Box sx={styles.rightContent}>
+      <Box sx={styles.searchBarContainer}>
+        <SearchBar placeholder="Search Chirp" />
+      </Box>
+
+      <Box sx={styles.suggestedUserContainer}>
```

Comment:
I think we can put this header into `SuggestedUsers`

Reply:
Moved reusable containers such as `SuggestedUsers` into components. This includes the `SidebarFooter` and `RelevantUsers`.